lib/lookup.json

```diff
@@ -457,25 +343,12 @@
     "envVar": { "npm_config_ignore_scripts": "true" },
     "comment": "postlint script fails"
   },
-  "serialport": {
```

Comment:
serialport is one of the few where we probably *don't* want to skip macOS if we can avoid it, but certainly skipping that one OS is better than complete removal. I'll add it back in and also remove the skip for Windows and run a CITGM. If it fails on Windows and/or macOS but nowhere else, we'll skip those but leave it alone otherwise. (Sorry our macOS test machines are not as robustly provisioned as we might like. Alas, we run on donated services, etc.)